### PR TITLE
updated-knowbe4

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [X] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by nmohantyctr — APPROVED** [review]
Have made changes to the table formatting for lines 110, 119, 124, and 125.

File: docs/search/search-cheat-sheets/index.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Cheat sheets provide examples of useful search queries for different use cases.
 
 For a collection of customer created search queries and their use cases, see the [Sumo Logic Community Query Library](https://support.sumologic.com/hc/en-us/community/topics/200396738-Query-Library). 
 
-For a step-by-step videos creating Sumo Logic queries, see the [Micro Lessons](https://[learn.sumologic.com](https://www.youtube.com/watch?v=1-f1X3Y9Wug&list=PLuHsjJUxgM1fRFUzFZuQcZ2GCW-jtiOxa)) or the [Tutorials](https://www.youtube.com/watch?v=_5JHkxG7ZMo&list=PLuHsjJUxgM1cmKQk1UjDUbiUcTnfgNwFl) .  
+For a step-by-step videos creating Sumo Logic queries, see the [Micro Lessons](https://www.youtube.com/playlist?list=PLuHsjJUxgM1fRFUzFZuQcZ2GCW-jtiOxa) or the [Tutorials](https://www.youtube.com/watch?v=_5JHkxG7ZMo&list=PLuHsjJUxgM1cmKQk1UjDUbiUcTnfgNwFl) .
```
Comment (kimsauce) [inline docs/search/search-cheat-sheets/index.md:13]:
```suggestion
For step-by-step videos on creating Sumo Logic queries, see our [Micro Lessons](https://www.youtube.com/playlist?list=PLuHsjJUxgM1fRFUzFZuQcZ2GCW-jtiOxa) and [Tutorials](https://www.youtube.com/watch?v=_5JHkxG7ZMo&list=PLuHsjJUxgM1cmKQk1UjDUbiUcTnfgNwFl).  
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/knowbe4-api-source.md [hunk 2]
```diff
@@ -1,14 +1,52 @@
...
+:::note
+To access the Knowbe4 API, you must have either the Platinum or Diamond user plan.
+:::
+
+## Prerequisites
+
+The KnowBe4 Source requires the following information to connect to KnowBe4 API:
+Before you begin setting up your **KnowBe4**** Source, the integration must be configured with the **Base URL** and **KnowBe4 API Token**.
+1. Base URL
+2. API Token
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/knowbe4-api-source.md:21]:
```suggestion
Before you begin setting up your **KnowBe4** Source, which is required to connect to KnowBe4 API, you'll need to configured your integration with the **Base URL** and **KnowBe4 API Token**.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/knowbe4-api-source.md [hunk 3]
```diff
@@ -1,14 +1,52 @@
...
+
+### Base URL
+
+The **Base URL** is the URL where your **KnowBe4** account is located  To get the base URL, follow the steps below:
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/knowbe4-api-source.md:25]:
```suggestion
The **Base URL** is the URL where your **KnowBe4** account is located. To get the base URL, follow the steps below:
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/knowbe4-api-source.md [hunk 4]
```diff
@@ -1,14 +1,52 @@
...
+  | :---|:---|:---|
+  | US Server |	training.knowbe4.com | `https://api.events.knowbe4.com` |
+  | EU Server | eu.knowbe4.com | `https://api-eu.events.knowbe4.com` |
+  | CA Server |	ca.knowbe4.com | `https://api-ca.events.knowbe4.com` |
+
+
+### API Token
+
+The **API security token** is used to authenticate with KnowBe4 HTTP API. To get the **KnowBe4 API token**, follow the steps below:
+1. Log into **KnowBe4** application.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/knowbe4-api-source.md:41]:
```suggestion
1. Log in to the **KnowBe4** application.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/knowbe4-api-source.md [hunk 5]
```diff
@@ -1,14 +1,52 @@
...
+1. Navigate to **Events** section from left panel and click **Settings**.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/knowbe4-api-source.md:42]:
```suggestion
1. Navigate to the **Events** section from left panel and click **Settings**.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/knowbe4-api-source.md [hunk 6]
```diff
@@ -50,30 +86,47 @@ To configure a KnowBe4 API Source:
...
 
-## API Limitations
 
-The number of licenced users on your account can make a maximum of ten requests per day using the KnowBe4 API. You may access the APIs only four times per second.
+### JSON configuration
+
+Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [how to use JSON to configure Sources](/docs/send-data/use-json-configure-sources) for details. 
+
+| Parameter | Type | Required | Description | Access |
+|--|--|--|--|--|
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/knowbe4-api-source.md:110]:
```suggestion
|:--|:--|:--|:--|:--|
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/knowbe4-api-source.md [hunk 7]
```diff
@@ -50,30 +86,47 @@ To configure a KnowBe4 API Source:
...
+| `config` | JSON Object  | Yes | Contains the [configuration-parameters](#config-parameters) of the Source. | not modifiable |
+| `schemaRef` | JSON Object  | Yes | Use `{"type":"KnowBe4"}` for KnowBe4 Source. | not modifiable |
+| `sourceType` | String | Yes | Use `Universal` for KnowBe4 Source. | not modifiable |
+
+
+### Config Parameters
+
+| Parameter | Type | Required | Description | Access |
+|---|---|---|---|---|
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/knowbe4-api-source.md:119]:
```suggestion
|:---|:---|:---|:---|:---|
```

**Pull request merged**